src/core/InstrumentFunctions.cpp

```diff
@@ -378,9 +378,7 @@ void InstrumentFunctionArpeggio::processNote( NotePlayHandle * _n )
 	// used for calculating remaining frames for arp-note, we have to add
 	// arp_frames-1, otherwise the first arp-note will not be setup
 	// correctly... -> arp_frames frames silence at the start of every note!
-	int cur_frame = ( ( m_arpModeModel.value() != FreeMode ) ?
```

Comment:
I'm dropping this test altogether but perhaps it should be kept and changed to `m_arpModeModel.value() = SyncMode` ?